src/Microsoft.DotNet.Build.Tasks.net45/CreateExternalAssembliesResources.cs

```diff
@@ -0,0 +1,78 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+using System.Collections;
+using System.IO;
+using System.Reflection;
+using System.Resources;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class CreateExternalAssembliesResources : Task
+    {
+        [Required]
+        public ITaskItem[] InputAssemblies { get; set; }
+
+        [Required]
+        public string OutputPath { get; set; }
+
+        public override bool Execute()
+        {
+            try
+            {
+                if (!Directory.Exists(OutputPath))
```

Comment:
you don't need the directory.exists check. it will just do nothing if it already exists